package.json

```diff
@@ -42,7 +43,6 @@
     "eslint-plugin-mocha": "^6.2.2",
     "eslint-plugin-prettier": "^3.1.2",
     "grpc-tools": "1.8.1",
-    "grpc_tools_node_protoc_ts": "2.5.10",
```

Comment:
Why did this dependency get dropped?

Reply:
@hbergren This dependency generated the typings for gRPC-Node output where as gRPC-Web does this automatically when using the `commonjs+dts` option in the generator.